packages/datagateway-search/src/table/investigationSearchTable.component.test.tsx

```diff
@@ -127,6 +127,7 @@ describe('Investigation Search Table component', () => {
     ];
     (useCart as jest.Mock).mockReturnValue({
       data: [],
+      isLoading: false,
```

Comment:
This line is fine, I just couldn't comment on the line I wanted.

Just for maintainability, in the `afterEach` on line 183, I'd change this all to `jest.clearAllMocks()` to make sure we're not missing anything, instead of calling `mockClear` line after line. If you ever come across any others like this, I'd do the same, it's just easier down the line